<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Best Painter - Paint</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="width_sb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="accelerated">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_color_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cur_player_lbl">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сейчас рисует:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Осталось времени:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_lcn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>820</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Заливка</string>
    </property>
   </widget>
   <widget class="QLabel" name="word_lbl">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
